Websites/bugs.webkit.org/template/en/custom/account/auth/login-small.html.tmpl

```diff
@@ -37,7 +37,7 @@
   <form action="[% login_target FILTER html %]" method="POST"
         id="mini_login[% qs_suffix FILTER css_class_quote %]" class="mini_login dialog-grid">
         
-    <input id="Bugzilla_login[% qs_suffix FILTER css_class_quote %]" required
+    <input id="Bugzilla_login[% qs_suffix FILTER css_class_quote %]" required autofocus
```

Comment:
`autofocus` here is redundant. The dialog focusing steps for modal dialogs should automatically pick the first focusable element and focus it.

Reply:
Right. However, the first focusable element that WebKit chooses is the "close" button, hence the need for the `autofocus` attribute on the login field.